src/pages/Results/pages/Results/pages/FootO/pages/Splits/components/FootOSplitsTable/components/RunnerRow.tsx

```diff
@@ -119,92 +115,140 @@ export default function RunnerRow(props: RunnerRowProps) {
       ? Math.max(0, result.time_seconds - totalLossTime)
       : 0
 
-  const handleSelectionChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    props.onSelectionChange?.(props.runner.id, event.target.checked)
-  }
-
   return (
-    <TableRow key={props.runner.id} sx={{ padding: "none" }}>
-      {props.graphsEnabled && (
-        <TableCell key="selection" padding="checkbox">
-          <Checkbox
-            checked={props.selected || false}
-            onChange={handleSelectionChange}
-            disabled={!props.canSelect || (props.maxRunnersReached && !props.selected)}
-            color="primary"
-            title={
-              !props.canSelect
-                ? "Solo corredores con estado OK pueden ser seleccionados"
-                : props.maxRunnersReached && !props.selected
-                  ? "Máximo de corredores alcanzado para este tipo de gráfico"
-                  : undefined
-            }
-          />
+    <React.Fragment>
+      <TableRow key={`runnerNameRow-${props.runner.id}`}>
+        <TableCell
+          key={`name${props.runner.id}`}
+          colSpan={splits.length + 1 + (props.timeLossEnabled && !props.showCumulative ? 1 : 0)}
+          sx={{
+            border: "none",
+            backgroundColor: "white",
+            fontWeight: "bold",
+            padding: `16px 16px calc(16px + 2rem) 16px`,
+          }}
+        >
+          <Box
+            className="wrapper"
+            sx={{ display: "inline-flex", position: "absolute", pointerEvents: "none" }}
+          >
+            <RacePosition
+              position={props.runner.stage.position}
+              isNC={props.runner.is_nc || status === RESULT_STATUS_TEXT.nc}
+              hasDownload={hasChipDownload}
+            />
+            <ParticipantName
+              name={props.runner.full_name}
+              subtitle={runnerService.getClubName(props.runner, t)}
+            />
+          </Box>
```

Comment:
Hacky new second row for each runner, where it's just a single cell with a position absolute to have the runner name, race position and stuff always visible even when scrolled. Not really happy with the solution, feels too hacky, I'll have to dig a bit when I extract the tightly coupled table component into a generic one